### PR TITLE
Update party-ui-handler.ts (IT)

### PR DESCRIPTION
**Pull request by NicusPulcis**
## What are the changes?
localized the whole file in italian
## Why am I doing these changes?
because it wasnt localized yer
## What did change?
everything in the party ui handler file has been translated in italian
## Checklist
- [x] **I'm using `beta` as my base branch**
- [ ] There is no overlap with another PR?
- [x] The PR is self-contained and cannot be split into smaller PRs?
- [x] Have I provided a clear explanation of the changes?
- [ ] Have I considered writing automated tests for the issue?
- [ ] Have I tested the changes (manually)?
    - [ ] Are all unit tests still passing? (`npm run test`)
- [x] Are the changes visual?
  - [ ] Have I provided screenshots/videos of the changes?

### CODE REVIEW

File: src/locales/it/party-ui-handler.ts [hunk 1]
```diff
@@ -7,48 +7,48 @@ export const partyUiHandler: SimpleTranslationEntries = {
...
-  "cancel": "Cancel",
+  "choosePokemon": "Scegli un Pokémon.",
+  "doWhatWithThisPokemon": "Hai selezionato questo Pokémon.",
+  "noEnergy": "{{pokemonName}} non ha più energie\nper lottare!",
+  "hasEnergy": "{{pokemonName}} ha ancora energie\nper lottare!",
+  "cantBeUsed": "{{pokemonName}} non può essere usato\nin questa sfida!",
+  "tooManyItems": "{{pokemonName}} possiede già\nquest'oggetto in abbondanza!",
+  "anyEffect": "Non avrebbe alcun effetto.",
+  "unpausedEvolutions": "{{pokemonName}} può di nuovo evolversi.",
+  "unspliceConfirmation": "Vuoi davvero dividere {{fusionName}}\nda {{pokemonName}}? {{fusionName}} andrà perduto.",
```
Comment (CodeTappert) [inline src/locales/it/party-ui-handler.ts:29]: Provide a screenshot to see if its not tool ong

Comment (NicusPulcis) [reply src/locales/it/party-ui-handler.ts:29]: hey uhmmm so i never tested stuff before and i don't know how to do that. @CodeTappert could you help me out with that?

**Review by EnochG1 — APPROVED** [review]

**Comment by AsdarDevelops** [comment]
I'm making a PR with this. You guys can close.

**Comment by DayKev** [comment]
Migrated to #3737

**Pull request closed without merging**